modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeCatchAllCheck.java

```diff
@@ -140,7 +140,7 @@ private static Pattern _getPattern(JSONObject jsonObject) {
 			regex = StringUtil.replace(from, CharPool.PERIOD, "\\.\\s*");
 		}
 
-		return Pattern.compile(regex + "\\(");
+		return Pattern.compile(regex + "[\\(;]");
```

Comment:
@nicolas-sss, can you please describe a little bit more the purpose of this change in the commit message?

Right now, it's too generic